decisions/dependencies.md

```diff
@@ -1,11 +0,0 @@
-# Dependencies
-
-The following dependencies aren't actively used by this repo but we need them for some other dependencies to work:
-
-* `acorn`
-* `util`
```

Comment:
- We actually are using `util` so this shouldn't be listed here.
- We're not using `corejs` anywhere and it's not listed as a dep.